pkg/virtctl/create/preference/preference.go

```diff
@@ -0,0 +1,202 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package preference
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"sigs.k8s.io/yaml"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	instancetypev1alpha2 "kubevirt.io/api/instancetype/v1alpha2"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/create/params"
+)
+
+const (
+	Preference = "preference"
+
+	CPUFlag        = "cpu"
```

Comment:
IMHO this could be more specific if we aren't allowing sub options for these switches. `cpu-topology`?